packages/MSBot/src/msbot-clone-services.ts

```diff
@@ -956,28 +958,44 @@ async function updateLocalSafeSettings(azBot?: IBotService): Promise<void> {
     }
 }
 
+/**
+ * Calls creates a publish command and potentially calls it if code-dir is passed in.
+ * Also creates local scripts to make it easier to publish local source code by already having the code formulated.
+ * @param azBot
+ */
 async function publishBot(azBot: IBotService): Promise<void> {
-    let azPublishCmd = `az bot publish --resource-group ${args.groupName} -n ${azBot.name} --subscription ${args.subscriptionId} -v ${args.sdkVersion || 'v4'} `;
-    if (args.verbose) {
-        azPublishCmd += '--verbose ';
-    }
-
+    let azPublishCmd: string = createPublishCmds(azBot);
     let result: string | null = null;
+    if (args.codeDir) {
+        azPublishCmd += `--code-dir "${args.codeDir}" `;
+        result = await runCommand(azPublishCmd, `Publishing the local folder ${args.codeDir} to ${args.name} service`);
+    }
+    await createLocalPublishCmds(azPublishCmd, !!result);
+}
 
+function createPublishCmds(azBot: IBotService): string {
+    let azPublishCmd = `az bot publish --resource-group ${args.groupName} -n ${azBot.name} --subscription ${args.subscriptionId} -v ${args.sdkVersion || 'v4'} `;
     if (args.projFile) {
         azPublishCmd += `--proj-name "${args.projFile}" `;
-        result = await runCommand(azPublishCmd, `Publishing the local project ${args.projFile} to ${args.name} service`);
-    } else {
-        azPublishCmd += `--code-dir "${args.codeDir}" `;
-        result = await runCommand(azPublishCmd, `Publishing the local folder ${args.codeDir} to ${args.name} service`);
     }
+    if (args.verbose) {
+        azPublishCmd += '--verbose ';
+    }
+    
+    return azPublishCmd;
+}
 
+async function createLocalPublishCmds(azCmd: string, didPublishOccur: boolean): Promise<void> {
+    if (!didPublishOccur) {
+        console.log(chalk.default.yellowBright(`Your local bot was not published to Azure.`));
```

Comment:
Why use ticks for strings here when you aren't templating anything? This is especially confusing because you don't use ticks when actually string templating on line 994.